<?xml version="1.0" encoding="UTF-8"?><Posts><Post><PostId>9570637</PostId><Url>http://stackoverflow.com/questions/9570637</Url><Stack><Type>ConfigurationError</Type><Message>Reserved characters such as ':' must be escaped according RFC 2396. An IPv6 address literal must be enclosed in '[' and ']' according to RFC 2732.</Message><Frame><File>/usr/local/lib/python2.7/dist-packages/celery/worker/__init__.py</File><Line>230</Line><Method>start</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/celery/worker/consumer.py</File><Line>338</Line><Method>start</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/celery/worker/consumer.py</File><Line>596</Line><Method>reset_connection</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/celery/app/amqp.py</File><Line>335</Line><Method>get_task_consumer</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/kombu/compat.py</File><Line>187</Line><Method>__init__</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/kombu/messaging.py</File><Line>285</Line><Method>__init__</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/kombu/messaging.py</File><Line>295</Line><Method>declare</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/kombu/entity.py</File><Line>388</Line><Method>declare</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/kombu/entity.py</File><Line>408</Line><Method>queue_declare</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/kombu/transport/virtual/__init__.py</File><Line>380</Line><Method>queue_declare</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/kombu/transport/mongodb.py</File><Line>74</Line><Method>_size</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/kombu/transport/mongodb.py</File><Line>171</Line><Method>client</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/kombu/transport/mongodb.py</File><Line>97</Line><Method>_open</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/pymongo/connection.py</File><Line>325</Line><Method>__init__</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/pymongo/uri_parser.py</File><Line>198</Line><Method>split_hosts</Method></Frame><Frame><File>/usr/local/lib/python2.7/dist-packages/pymongo/uri_parser.py</File><Line>127</Line><Method>parse_host</Method></Frame></Stack></Post></Posts>